testing/internal-testing/src/main/groovy/org/gradle/integtests/fixtures/HtmlTestExecutionResult.groovy

```diff
@@ -227,6 +229,25 @@ class HtmlTestExecutionResult implements TestExecutionResult {
             return this
         }
 
+        @Override
+        TestClassExecutionResult assertTestSkipped(String name, Consumer<SkippedExecutionResult> assertions) {
+            TestCase testCase = testsSkipped.find { it == testCase(name) }
+            assert testCase
+
+            final SkippedExecutionResult skippedExecutionResult
+            if (!testCase.messages.isEmpty()) {
+                String possibleText = testCase.getMessages().first()
+                String[] stacktraceMessage = possibleText.readLines().first().split(":", 2)
+                assert stacktraceMessage.length == 2 // sanity check that we're parsing something that looks like an exception stacktrace
```

Comment:
🤔 This seems like a poor sanity check.  If we really care about this, we can surely do something more accurate.